src/gc/gc.cpp

```diff
@@ -899,39 +893,69 @@ class t_join
         }
     }
 
+#ifdef JOIN_STATS
+    uint64_t getTS()
+    {
+        return GCToOSInterface::QueryPerformanceCounter();
```

Comment:
please try to conform to GC's coding style - rename getTS to something like get_ts or get_current_time and starts accordingly.